docs/tempo/website/getting-started/quickstart-tempo.md

```diff
@@ -1,83 +0,0 @@
----
```

Comment:
Deleted by mistake? This page has links from other places.. and we don't have a doc that covers how we can start sending existing traces to Tempo

Reply:
I removed this b/c the information seemed redundant with the examples in the repo so I just redirected to that page. Also, I feel like we will keep the examples updated, but it would be easy to let this page lag behind.

I can be persuaded otherwise if we feel like this would be a good page to keep.